models/ac_generator.h

```diff
@@ -128,12 +164,20 @@ class ac_generator : public Node
     double A_11_;
   };
 
+  double
+  get_I_() const
+  {
+    return S_.I_;
```

Comment:
I tried incorporating the changes for AC and DC (without S_.I_). This is what I got:
![output_all_bad](https://cloud.githubusercontent.com/assets/24866517/24666191/e6f79d4a-195f-11e7-8871-3271aa611147.png)